utils/release.bash

```diff
@@ -39,18 +39,20 @@ function configureGit() {
 function configureGPG() {
 	requireGPGPassphrase
 	if ! gpg --fingerprint "${GPG_KEYNAME}"; then
-		if [ ! -f "${GPG_FILE}" ]; then
+		if [[ ! -f $GPG_FILE ]]; then
```

Comment:
Is there a reason to remove the quotes? If the variables `GPG_FILE` contains a special character such as a space, then the instruction would fail.

Reply:
I thought so too at first, but then I learned that this is not the case when using Bash's fancy double-bracket conditionals, as is being done here. But this would definitely be a concern in non-Bash shells, or even in Bash when using single-bracket conditionals. So this is why `shfmt -s` strips the quotes in this case.

Reply:
Thanks for the feedback! Really interesting to know, I need to read more on this.